source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -485,7 +487,14 @@ removed once the primary is checked.
 #### error
 
 If the client experiences any error when checking a server, it stores error information in the ServerDescription's error
-field.
+field. The message contained in this field MUST contain the substrings detailed in the table below when the
+ServerDescription is changed to Unknown in the circumstances outlined.
+
+| circumstance                                                         | error substring                                          |
+| -------------------------------------------------------------------- | -------------------------------------------------------- |
+| RSPrimary with a stale electionId is discovered                      | `'primary marked stale due to electionId mismatch'`      |
+| RSPrimary with a stale setVersion is discovered                      | `'primary marked stale due to setVersion mismatch'`      |
+| A more current RSPrimary is discovered alongside an existing primary | `'primary marked stale due to discovery of new primary'` |
```

Comment:
Should we combine this case as well? IIUC the case is the same as the above but just from the other server's point of view.